src/app/lib/utilities/ichefURL/index.js

```diff
@@ -2,28 +2,23 @@
 const WEBP_ORIGIN_CODES = ['cpsdevpb', 'cpsprodpb'];
 
 const buildPlaceholderSrc = (src, resolution) => {
-  const parts = src.split('/');
-  const [domain, media, imgService, width, ...extraParts] = parts;
-  const definedWidth = width.replace('$width', resolution);
-  const domainWithProtocol = `https://${domain}`;
-
+  if (src.includes('urn:') || src.includes('localhost:')) return src;
+  const urlParts = src.replace(/https:\/\/|http:\/\//, '').split('/');
```

Comment:
I think this can be shortened to:

```suggestion
  const urlParts = src.replace(/https?:\/\//g, '').split('/');
```